app/templates/application.hbs

```diff
@@ -14,3 +14,12 @@
 {{/init-foundation}}
 {{outlet}}
 <div class="printer_message_block"></div>
+
+{{!-- Service-backed components --}}
+{{#let-alias designationService as |s|}}
+  {{goodcity/orders-search-overlay
+    open=s.openOrderSearch
+    onSelect=s.onnOrderSelected
```

Comment:
`onn` typo?

Reply:
Darn, my brokenn Mac keyboard keeps doubling the Ns and the Ts